### ISSUE TITLE
Screen reader pronounces nitric oxide as "no" and nitrogen dioxide as "number 2"

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#255]
For https://github.com/phetsims/QA/issues/461. Noticed on iPadOS 13.3 VO.
When reading out "NO2" the screen reader pronounces this as "number 2" instead of "N" "O" "2."  When UV light breaks the molecule apart it reads NO as "no."

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe, this is unfortunate. I think we were aware of this but it is good to have documentation. To my knowledge there is no way to force a screen reader to pronounce things differently. We could use a string like "N O" but don't know if we should be doing that.

@terracoda can you please comment on if anything should be done about this?

**Comment by terracoda** [comment]
We will work on how best to represent Chemical abbreviations as we continue to work on this sim.

**Comment by jessegreenberg** [comment]
In https://github.com/phetsims/molarity/issues/215 it was decided that there isn't anything we can do about how the screen reader reads abbreviations.
